group_vars/model_avm_fritzbox_7530_dsa.yml

```diff
@@ -0,0 +1,27 @@
+---
+openwrt_version: snapshot
+target: ipq40xx/generic
+override_target: "avm_fritzbox-7530"
+
+model__packages__to_merge:
+  - "-kmod-ath10k-ct -ath10k-firmware-qca4019-ct"
+  - "kmod-ath10k ath10k-firmware-qca4019"
+
+dsa_ports:
+  - lan1
+  - lan2
+  - lan3
+  - lan4
+  - wan
```

Comment:
the 7530 does not have a wan port? or are you talking about the dsl line?

Reply:
Oops, you're right, I also removed the wan port from the old model file in a second commit.